### PR TITLE
options for hiding y axis, hiding track selector, set plot height

### PR DESCRIPTION
**Pull request by leexgh**
implement https://github.com/cBioPortal/cbioportal/issues/6618

### CODE REVIEW

File: src/MutationMapper.tsx [hunk 1]
```diff
@@ -31,6 +31,8 @@ export type MutationMapperProps = {
     windowWrapper?: {size: {width: number, height: number}};
     trackVisibility?: TrackVisibility;
     tracks?: TrackName[];
+    showTrackSelector?: boolean;
+    showYAxis?: boolean;
```
[the remark this answers is not in the document]
Comment (leexgh) [reply src/MutationMapper.tsx]: @onursumer you want all change to `showPlotYAxis` or just the props in MutationMapper?

[the remark this answers is not in the document]
Comment (onursumer) [reply src/MutationMapper.tsx]: @leexgh just in `MutationMapper`

[the remark this answers is not in the document]
Comment (leexgh) [reply src/MutationMapper.tsx]: got it

File: src/MutationMapper.tsx [hunk 2]
```diff
@@ -31,9 +31,11 @@ export type MutationMapperProps = {
     windowWrapper?: {size: {width: number, height: number}};
     trackVisibility?: TrackVisibility;
     tracks?: TrackName[];
+    showTrackSelector?: boolean;
     customMutationTableColumns?: DataTableColumn<Partial<Mutation>>[];
     customMutationTableProps?: Partial<TableProps<Partial<Mutation>>>;
     showFilterResetPanel?: boolean;
+    showPlotAxis?: boolean;
```
Comment (onursumer) [inline src/MutationMapper.tsx]:
```suggestion
    showPlotYAxis?: boolean;
```

File: src/MutationMapper.tsx [hunk 3]
```diff
@@ -258,19 +261,22 @@ export default class MutationMapper<P extends MutationMapperProps = MutationMapp
...
                 trackDataStatus={this.trackDataStatus}
+                showTrackSelector={this.props.showTrackSelector}
                 onXAxisOffset={this.onXAxisOffset}
                 onTrackVisibilityChange={this.props.onTrackVisibilityChange}
                 getMutationCount={this.props.getMutationCount}
                 getLollipopColor={this.props.getLollipopColor}
                 yMaxLabelPostfix={this.plotYMaxLabelPostfix}
                 yMaxFractionDigits={this.props.plotYMaxFractionDigits}
                 yAxisLabelPadding={this.props.plotYAxisLabelPadding}
+                showYAxis={this.props.showPlotAxis}
```
Comment (onursumer) [inline src/MutationMapper.tsx]:
```suggestion
                showYAxis={this.props.showPlotYAxis}
```

**Pull request merged**